### PR TITLE
Template update 2.11

### PR DESCRIPTION
**Pull request by danilodileo**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/metatdenovo/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/metatdenovo _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: nextflow_schema.json [hunk 1]
```diff
@@ -229,7 +229,7 @@
                 "prokka_batchsize": {
                     "type": "string",
                     "description": "Size of individual files annotated by Prokka in one batch.",
-                    "default": "10.MB",
+                    "default": "10 MB",
```
Comment (erikrikarddaniel) [inline nextflow_schema.json:232]: Is this correct? On the command line you can't say `10 MB` without using quotes.

**Review by erikrikarddaniel — APPROVED** [review]
The "TODO string in awsfulltest.yml: You can customise AWS full pipeline tests as required" warning -- is that something we can close now?

One more tiny comment.

File: nextflow_schema.json (hunk 1 above)
Comment (danilodileo) [reply nextflow_schema.json:232]: I think it's a change made by prettier or lint, I didn't change it. Anyway I didn't get any warning about it.

**Comment by danilodileo** [comment]
> The "TODO string in awsfulltest.yml: You can customise AWS full pipeline tests as required" warning -- is that something we can close now?
> 
> One more tiny comment.

We did that so I removed the TODO

**Pull request merged**